src/effects/native/nativebackend.h

```diff
@@ -6,7 +6,7 @@
 class NativeBackend : public EffectsBackend {
     Q_OBJECT
   public:
-    NativeBackend(QObject* pParent=NULL);
+    NativeBackend(UserSettingsPointer pConfig, QObject* pParent = nullptr);
```

Comment:
Conventionally the `QObject* pParent` argument to a constructor is the first argument.